### PR TITLE
Update Vague.js license info in package.json

### PR DESCRIPTION
**Pull request by jeremywu3**
Update it because its license is "MIT"

### CODE REVIEW

File: ajax/libs/Vague.js/package.json [hunk 1]
```diff
@@ -26,4 +26,5 @@
     "type": "git",
     "url": "git://github.com/GianlucaGuarini/Vague.js.git"
   }
```
Comment (extend1994) [inline ajax/libs/Vague.js/package.json]: Please add a comma(,) here to pass the test.

Comment (sufuf3) [reply ajax/libs/Vague.js/package.json]:
```
   }
```
to 
```
   },
```

File: ajax/libs/Vague.js/package.json [hunk 2]
```diff
@@ -26,4 +26,5 @@
...
+  "license": "MIT";
```
Comment (maruilian11) [inline ajax/libs/Vague.js/package.json]: Please modify the change to :
```
  "repository": {
    "type": "git",                                                                                                                           
    "url": "git://github.com/GianlucaGuarini/Vague.js.git"
  },
  "license": "MIT"
```

**Comment by maruilian11** [comment]
Please remember to provide the license ref next time, thanks.

**Comment by sashberd** [comment]
@PeterDaveHello This one was already fixed https://github.com/cdnjs/cdnjs/commit/44c9758a11d18c020646b6cac3d2abc37fe7f3a8

**Pull request closed without merging**